akirachix/Magezi-Informational
# Feature/insights

# Engineers Checklist
This pull request updates the About Us and Insights sections to improve clarity and engagement, addressing feedback from users about the existing content.
### Motivation and Context.
 The motivation for these updates is to enhance user understanding of our mission and the insights we provide. Feedback indicated that the previous content was unclear and did not effectively convey our value proposition.
List any dependencies that are required for this change.

## What this PR does?
Perhaps answering the following questions will help create a good description:
### What is this?
This PR revises the About Us section to better articulate our mission and updates the Insights section to include more recent data and findings.
###  Why this?
This is necessary to ensure our users have accurate and engaging information that reflects our current direction and expertise.
### How are you doing this? 
Updated the About Us text for clarity and conciseness.
Revised the "Insights" section to include new statistics and analysis relevant to our audience.
## List of updates for features, enhancements, or bugfixes
Improved the language and structure of the About Us section.
Added new insights and data visualizations to the Insights section.

## Type of change

Specify the kind of PR/MR

Please delete options that are not relevant.


## How can it be tested?
- Navigate to the "About Us" page and review the updated content.
Go to the "Insights" section and verify the new data and visualizations.
Ensure that all links and references are functioning correctly.
# Where is it documented?
- List JIRA tickets
- https://nakatozainamusana.atlassian.net/jira/software/projects/SCRUM/boards/4/backlog?assignee=unassigned%2C712020%3Af1462196-a595-4f92-9d3d-1680ff7a9a3c&atlOrigin=eyJpIjoiZDQxNGJkN2E2ZWUyNGNlZThhNWY2ZTdlYWU0ZjA3NGUiLCJwIjoiaiJ9
